royalroad.params

```diff
@@ -1,18 +1,19 @@
-Experiment ID                :onemax
-Problem Type                 :OM
+Experiment ID                :royalroad
+Problem Type                 :RR
+Varient                      :RR1
 Data Input File Name         :NA
-Number of Runs               :10
-Generations per Run          :200
-Population Size              :100
+Number of Runs               :100
+Generations per Run          :10
```

Comment:
10 generations seems low, doesn't it?

Reply:
I thought it mentioned 10 in the paper which is why I put that there, but yeah it’s very low